### ISSUE TITLE
...

### ISSUE THREAD

**Issue by tonywied17** [CpanelInc/ea-nodejs16#1]
Can we get more support for node js developers?
CPanel seems to still be focused on PHP driven development in 2023.

Either stay up to date with node js and npm releases for easy apache or have a better way of support node js.

For the premium price of cpanel licensing other panels seem to do node js better like aapanel which is literally free...

**Comment by gitcory** [comment]
Are you asking for like nodejs18/19 or asking that nodejs16 be updated? 16.20.1 is the lastest version currently.

**Comment by tonywied17** [comment]
Yeah just better overall support for releases. Or maybe a built in node version manager with a better ui for configuring apps/deployments.

Also worth noting I feel like the process to deploy a node js application is weird with the whole passenger startup file process.
Just feels like some outdated hackery is implemented when it comes to node js support for cpanel.

I did try the Cloud Linux addon for a while and that had a better node js app deployment module but still had outdated versions. Idk for me its the one thing that cpanel is missing.
